### DISCUSSION TITLE
What's new in Prism 8 and 9

### DISCUSSION THREAD

**Discussion by sylvesp** [PrismLibrary/Prism#3098] [General] [answered]
I may have posted this question originally to the wrong place on the internet :o (stackoverflow). 
Hopefully this is the right place to post it.
-----------------------------------------------------------------
Our company has been using Prism 7.x for a few years now. There is a 'big migration/refactoring' going to happen and I was wondering if is there a "What's new?" document for Prism 8 and 9?

I did try to find it on the different Prism resource sites but had not much luck :(

I would be also interested in finding out:
- Which version of Prism did/will add support for Microsoft.Extensions.DependencyInjection? Is it only the Comercial Pluss v.9?
- Is Prism 9 (prerelease version if I understand correctly) 'mature enough' to be used in production software?

Knowing these details is important in our consideration to go for (quite a few) Comercial License(s).
Would appreciate your help.

Peter.

**Comment by dansiegel** [comment, marked as answer]
As for "whats new" I would suggest reading the release notes. As for the Microsoft Extensions Dependency Injection this is only available in Prism 9 and only for Commercial Plus license holders. 

As for when it will be available I cannot say exactly but I will say that it should be this month.
